File: HPWorkpath_v1.6.3/Samples/ExampleAPIServices/source/EmailSample/src/com/hp/workpath/sample/emailsample/EmailDialog.java
// Copyright 2018 HP Development Company, L.P.
// SPDX-License-Identifier: MIT
package com.hp.workpath.sample.emailsample;

public class EmailDialog {
    public static final String DIALOG_TYPE = "dialog_type";

    public enum Type {
        ADD_TO, ADD_CC, ADD_BCC, ATTACH, SMTP, PROXY
    }
}

File: HPWorkpath_v1.6.3/Samples_Kotlin/ExampleAPIServices/source/EmailSample/src/com/hp/workpath/sample/emailsample/EmailDialog.kt
// Copyright 2018 HP Development Company, L.P.
// SPDX-License-Identifier: MIT
package com.hp.workpath.sample.emailsample

object EmailDialog {
    const val DIALOG_TYPE = "dialog_type"

    enum class Type {
        ADD_TO, ADD_CC, ADD_BCC, ATTACH, SMTP, PROXY
    }
}